contact-us.html

```diff
@@ -11,5 +11,5 @@ <h1>Phone, Email or Visit</h1>
     <p>If you would like to visit the school, our office is open from 9:00am to 5:00pm Monday through Thursday, 9:00am until 1:00pm on Friday, and 9:30am until 12:00pm on Saturday. The school is open for classes from 7:30am to 9:00pm Monday through Thursday, 7:30am until 6:00pm on Friday, and 7:30am until 5:00pm on Saturday.</p>
     <p>English Now! is located at <strong>4963 Elm Street, Suite 300</strong>, Bethesda, Maryland 20814. We are located near shopping, the Connie Morella (Bethesda) Library, and many restaurants. We are at the corner of Elm Street and Arlington Road, across the street from the Giant grocery store. We are a few miles north of Washington, D.C. We are just three blocks from the Bethesda Metro Station on the Metro red line, and buses at the Bethesda Metro Station.</p>
     <p>Our building has two entrances, at <strong>4963 Elm Street</strong> and <strong>4940 Hampden Lane</strong>. The Elm Street entrance is on a busy shopping street. Most students and visitors will use the Elm Street entrance. First-time visitors arriving by car should use the Hampden Lane entrance. There is very convenient, free parking for 30 minutes, in a parking lot located next door at 4928 Hampden Lane, Bethesda MD 20814. From this parking lot walk 50 feet towards Arlington Road, and take the elevator at the Hampden Lane entrance up to the school on the third floor. The Hampden Lane entrance should also be used by students and visitors needing an elevator to access the third floor, or if arriving with a baby stroller. It is also the closest entrance to the Metro and bus station when walking.</p>
-    <p>Please feel free to call us at +1 301 718 3575 or email us at any time at <a href="" onclick="emailMe('info')">info@english-now.com</a></p> with any questions. We will respond as soon as possible.</p>
+    <p>Please feel free to call us at <a href="tel://+13017183575">+1 301 718 3575</a>, contact us by WhatsApp at the same number, or email us at <a href="" onclick="emailMe('info')">info@english-now.com</a> with any questions. We will respond as soon as possible.</p>
```

Comment:
I don't think you need the // I think it can just be tel:number

Reply:
Kinda like mailto:
https://www.elegantthemes.com/blog/wordpress/call-link-html-phone-number